recipes/kmc-smudge-pairs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "kmc_smudge_pairs" %}
+{% set version = "3.1.2rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gkanogiannis/KMC/archive/v{{ version }}.tar.gz
+  sha256: d6acc23643e3764a5ce6e3ec4cd1790592f36d992e44dc927baf0cbf14842f46
+
+build:
+  skip: true  # [not linux]
+  script: bash build_static.sh
```

Comment:
We would like to have shared builds on conda-forge.

Reply:
Hi @xhochy, thank you for reviewing this. I changed the code and makefile to build by linking to shared libraries (instead static .a provided for zlib and bzip2). The problem now is with the docker build that lacks these libraries. I get error "fatal error: zlib.h: No such file or directory". Any advice how to overcome this? Thank you for your support.